odonto/pathways.py

```diff
@@ -103,7 +112,7 @@ def save(self, data, user=None, patient=None, episode=None):
     pathway.Step(model=models.CaseMix, help_template="pathway/case_mix_help.html"),
 )
 
-
+#FP17
```

Comment:
These are file section marker comments? I don't think they actually enclose / mark the start and end of FP17 / FP17O in this file...

Reply:
Sure, I had actually reordered the file but it made the diff a bit unreadable so I've removed them for the time being.